modules/utils/docker/bridge_settings.sh

```diff
@@ -50,20 +50,111 @@ if [ -f "$COMMS_PCB_VERSION_FILE" ]; then
 fi
 echo $eth_port
 
-# Set mtu back to 1500 to support e2e connectivity
-# TODO: Investigate this as it should still work with 1460
-ifconfig bat0 mtu 1500
-ifconfig br-lan mtu 1500
-
-brctl addif br-lan bat0 $eth_port
-echo $br_lan_ip
-ifconfig br-lan $br_lan_ip netmask "255.255.255.0"
-ifconfig br-lan up
-echo
-ifconfig br-lan
+create_dhcpd_config()
```

Comment:
Generally we now have duplicates for these functions (create_dhcpd_config(), create_olsrd_config(), and create_radvd_config()) with mcc_settings.sh. I would like to see an "qos-olsr_config.sh" where these now go and then we have a "main()" function which calls them. You can then just call that bash script from BOTH mcc_settings.sh and bridge_settings.sh

Reply:
Fully agree :) but there are many other issues with HOS1.0 shell scripts anyway, would it make more sense to go straight to Python with unit testing etc? We'll likely need to share these create_dhcpd_config() and created_radvd_config() across even more modes so it might make sense to have them as dedicated classes in Python for example, or at least separate dhcpd.sh and radvd.sh files... These two being only two examples, there are probably even more topics (hostapd? wpa_supplicant?) that could benefit from being refactored into separate files/classes :)

Reply:
Sure, can you create an issue in github? We can then track it and pull it in later.